controller/src/main/java/io/pravega/controller/server/health/GRPCServerHealthContributor.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.controller.server.health;
+
+import io.pravega.controller.server.rpc.grpc.GRPCServer;
+import io.pravega.shared.health.Health;
+import io.pravega.shared.health.Status;
+import io.pravega.shared.health.impl.AbstractHealthContributor;
+
+public class GRPCServerHealthContributor extends AbstractHealthContributor {
+
+    private final GRPCServer grpcServer;
+
+    public GRPCServerHealthContributor(String name, GRPCServer grpcServer) {
+        super(name);
+        this.grpcServer = grpcServer;
+    }
+
+    @Override
+    public Status doHealthCheck(Health.HealthBuilder builder) throws Exception {
+        boolean running = grpcServer.isRunning();
+        Status status = Status.DOWN;
```

Comment:
nit: This can also be written like
`
Status status = running? Status.Up: Status.DOWN;

return status;
`

Reply:
Please do not resolve conversations unless you have responded to them or resolved them to the reviewers satisfaction.

Reply:
@shshashwat have refactored the code here, Please let me know if it is fine